File: POO/CS/OzCorp/OzCorp/Business/Manager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzCorp.Business
{
    public class Manager : Employee
    {
        public Manager(string Name, string RG, long CPF, string Enrollment, string Email, string Password, BloodType Blood, Gender Gender) : base(Name, RG, CPF, Enrollment, Email, Password, Role.GetRole(Roles.Manager), Blood, Gender)
        {
        }
    }
}

File: POO/J/OzCorp/src/br/com/ozcorp/business/Manager.java
package br.com.ozcorp.business;
public class Manager extends Employee {
	public Manager(String name, String rg, long cpf, String enrollment, String email, String password,
			BloodType blood, Gender gender) {
		super(name, rg, cpf, enrollment, email, password, Role.Manager, blood, gender);
	}
}
